apps/web/app/future/apps/[slug]/[...pages]/page.tsx

```diff
@@ -65,7 +64,7 @@ export const generateMetadata = async ({
 
   if (mainPage === "forms") {
     return await _generateMetadata(
-      () => `Forms | ${APP_NAME}`,
```

Comment:
`| ${APP_NAME}` should not be added here. It should be rather added in `preparePageMetadata`.